olive/workflows/run/config.py

```diff
@@ -50,7 +50,7 @@ def create_engine(self):
         return Engine(config)
 
 
-INPUT_MODEL_DATA_CONFIG_NAME = "olive_input_model_data_config"
+INPUT_MODEL_DATA_CONFIG = "INPUT_MODEL_DATA_CONFIG"
```

Comment:
what about use something start with double underscore? for example "__input_model_data_config__?

Reply:
Do they also provide the underscore value? or the constant name?

Reply:
It is the value. But I wasn't sure if we want user to provide the underscore value or constant name if we change it to underscore.

Reply:
IMHO, The constant name is Olive internally and user need only knowthe reserved names itself.

Reply:
That sounds good to me. I will update this. Thanks!

Reply:
Done! 
There are no documentation at all about data_configs so I think we can add the documentation in a later PR. Looks like it was not added when the concept was first introduced.